plugins/MLTModule.cpp

```diff
@@ -44,6 +44,28 @@ MLTModule::MLTModule(const std::string& name)
   // clang-format on
 }
 
+
+std::map<std::string, int>
+MLTModule::decode_geoid(uint64_t _geoid_int)
```

Comment:
It may have gone away with v5, but I thought we had GeoID decoding utilities somewhere else...

Reply:
The functions I used in v4 for that seemed to have disappeared in v5. 

Moreso, GeoID is now encoded into one long integer, rather than being an object. I found two separate (and identical) "encode" functions to make these, but no "decoding" function, which is what I need here... 
It definitely would be good to have them somewhere more reasonable than in the trigger.